packages/shared/config/src/enums.ts

```diff
@@ -71,7 +71,32 @@ export enum PORTFOLIO_CATEGORIES {
   MONEY_MARKET = 'money-market',
 }
 
-export enum SIDEBAR_CATEGORIES {
+export enum SETTINGS_SIDE_DRAWERS {
+  SETTINGS = 'settings',
+  NOTIFICATIONS = 'notifications',
+  CONNECT_WALLET = 'connect-wallet',
+}
+
+export enum SIDE_DRAWERS {
```

Comment:
Instead of copying and pasting these strings in two places, use a const instead for the SIDE_DRAWERS enum.
https://github.com/microsoft/TypeScript/issues/17592#issuecomment-320028196

```
enum BasicEvents {
    Start = "Start",
    Finish = "Finish"
};

// extend enum using "extends" keyword
const AdvEvents = {
    ...BasicEvents,
    Pause: "Pause",
    Resume: "Resume"
};
```

Reply:
This does not work as expected